### ISSUE TITLE
EA games folder requires administrator privileges

### ISSUE THREAD

**Issue by Lorin-lab** [R2NorthstarTools/FlightCore#241]
I tried to install Northstar with FlightCore but the folder `C:\Program Files\EA Games\Titanfall2` is not writable by a non-admin user.

The installation process does not result in an error but runs continuously.

I gave myself full access to the file and everything worked fine.

**Comment by GeckoEidechse** [comment]
Yeah, the lack of error message is cause the thread actually crashed instead of catching the error. A fix is already finished, just not published (should probably do that now actually).

> I gave myself full access to the file and everything worked fine.

Can you maybe expand a bit on what you did exactly? Did you run FlightCore as admin or did you do something else? ^^

**Comment by Lorin-lab** [comment]
> Can you maybe expand a bit on what you did exactly?

I juste change the folder permissions to give full control to all users.

Now if i try to run it as admin (from a non-admin user) I get this strange message:
![Screenshot 2023-03-28 185938](https://user-images.githubusercontent.com/72334359/228341048-af8046e1-734b-43ef-83c5-0d83cc5818af.png)

**Comment by GeckoEidechse** [comment]
> > Can you maybe expand a bit on what you did exactly?

> I juste change the folder permissions to give full control to all users.

As far as I know from others from testing that will always just reset itself. The recommended solution is to move the game install outside of `Program Files` entirely. IIRC the wiki recommends e.g. setting up something like `C:\Games\EA\Titanfall2` or similar.

> Now if i try to run it as admin (from a non-admin user) I get this strange message:
> ![Screenshot 2023-03-28 185938](https://user-images.githubusercontent.com/72334359/228341048-af8046e1-734b-43ef-83c5-0d83cc5818af.png)

That's the first time I'm seeing this, so kinda no clue.

For reference FlightCore uses WebView2 which is more or less a thin version of Edge, so that's where the Microsoft Edge part comes from. As to why it cannot write to that directory, idk tbh. Not even sure what that folder contains, i.g. you can try deleting it manually and then running FlightCore and if it doesn't launch, just reinstalling FlightCore. Settings are stored in `%appdata%\com.github.r2northstartools.flightcore` so you won't lose anything and it only stores like install path and release channel selection which is easy to re-create ^^
